_episodes/01-shell-basics.md

```diff
@@ -460,71 +452,70 @@ to edit the command line text
-                | (as defined by `VISUAL` environment variable)
+`Ctrl-x Ctrl-e` | (or `Alt-e` in **zsh**) -- Edit command line text in the editor (as defined by `VISUAL` environment variable)
 
 
 In addition to ability to edit command line text, some shortcuts
-control execution of the processes:
+control execution of processes:
 
 `Ctrl-c` | Kill currently running process
 `Ctrl-d` | Exit current shell
 `Ctrl-z` | Suspend currently running process. `fg` restores it, and `bg` would place it into background execution.
 
 > ## set -o
 >
-> Shells provide a set of configurable options which could be enabled or
-> disabled using `set` command.  Use `set -o` to print current settings
+> Shells provide a set of configurable options which can be enabled or
+> disabled using the `set` command.  Use `set -o` to print the current settings
 > you have in your shell, and then navigate `man bash` to find their
-> extended description (in `man` search by using shortcut `/` and
-> typing `o option-name`, and possibly 'n' for the "next" and 'p' for
-> "previous" finding to identify corresponding section.
-> E.g., use `set -o noclobber` which could be  recommended to forbid
-> overwriting previously existing files. `>|` could be used to
-> explicitly instruct overwriting already existing file. "Shell
+> extended description.
+>
+> You do so in `man` search by using the shortcut `/` and
+> typing `o option-name`. You can type 'n' for the "next" and 'p' for
+> "previous" finding to identify the corresponding section.
+> For example, use `set -o noclobber` which can be used to forbid
+> overwriting of previously existing files. `>|` could be used to
+> explicitly instruct the overwriting of an already existing file. “A shell
 > redirect ate my results file" should no longer be given as an excuse.
 >
 {: .challenge}
 
 ### Shell history
 
-By default, a shell stores in memory a history of the commands you
-have run.  You could access it using `history` command.  When you exit
+By default, a shell stores, in memory, a history of the commands you
+have run.  You can access it using the `history` command.  When you exit
 the shell, those history lines are appended to a file (by default in 
 `~/.bash_history` for bash shell). This not
 only allows you to quickly recall commands you have run recently, but
 can provide you an actual "lab notebook" of the actions you have
-performed. Thus the shell history could be indispensable to
-
-- provide a skeleton for your script and soon you realize that automating
-  current operations is worthwhile the effort, and
-- determine exactly what command you have run to perform some given
-  operation.
+performed. Thus the shell history can be very useful, as it can provide
+a skeleton for your script. When you soon realize that automating
+your current operations is worth the effort, you can determine exactly
+what command you have run to perform any given operation.
```

Comment:
well -- joining like that confuses the meaning of the 2nd item into the first. Even without writing a script your could use history to determine which command you ran. Please retain the items or rephrase/split to make it clear